Native compilation: Detected an instance of Random/SplittableRandom class in the image heap - java.security.SecureRandom

### Describe the bug

When i try to Native compile my Application i got this error.

```
Fatal error: org.graalvm.compiler.debug.GraalError: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  To see how this object got instantiated use --trace-object-instantiation=java.security.SecureRandom. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
        at com.oracle.graal.pointsto.util.AnalysisFuture.setException(AnalysisFuture.java:49)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:269)
        at com.oracle.graal.pointsto.util.AnalysisFuture.ensureDone(AnalysisFuture.java:63)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.lambda$postTask$9(ImageHeapScanner.java:611)
        at com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
        at com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
        at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  To see how this object got instantiated use --trace-object-instantiation=java.security.SecureRandom. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
        at com.oracle.svm.hosted.image.DisallowedImageHeapObjectFeature.error(DisallowedImageHeapObjectFeature.java:173)
        at com.oracle.svm.core.image.DisallowedImageHeapObjects.check(DisallowedImageHeapObjects.java:62)
        at com.oracle.svm.hosted.image.DisallowedImageHeapObjectFeature.replacer(DisallowedImageHeapObjectFeature.java:149)
        at com.oracle.graal.pointsto.meta.AnalysisUniverse.replaceObject(AnalysisUniverse.java:582)
        at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.replaceObject(AnalysisConstantReflectionProvider.java:257)
        at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.interceptValue(AnalysisConstantReflectionProvider.java:228)
        at com.oracle.svm.hosted.heap.SVMImageHeapScanner.transformFieldValue(SVMImageHeapScanner.java:126)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.onFieldValueReachable(ImageHeapScanner.java:331)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.onFieldValueReachable(ImageHeapScanner.java:310)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.lambda$computeTypeData$1(ImageHeapScanner.java:153)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        ... 10 more
```

After run 
`mvnw package -Pnative -Dquarkus.native.additional-build-args="--trace-object-instantiation=java.security.SecureRandom"`
 I get the following StackTracke
```
Fatal error: org.graalvm.compiler.debug.GraalError: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  Object has been initialized by the com.sap.conn.rfc.engine.GUID class initializer with a trace:
        at java.security.SecureRandom.<init>(SecureRandom.java:224)
        at com.sap.conn.rfc.engine.GUID.<clinit>(GUID.java:38)
. Try avoiding to initialize the class that caused initialization of the object. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
        at com.oracle.graal.pointsto.util.AnalysisFuture.setException(AnalysisFuture.java:49)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:269)
        at com.oracle.graal.pointsto.util.AnalysisFuture.ensureDone(AnalysisFuture.java:63)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.lambda$postTask$9(ImageHeapScanner.java:611)
        at com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
        at com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
        at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  Object has been initialized by the com.sap.conn.rfc.engine.GUID class initializer with a trace:
        at java.security.SecureRandom.<init>(SecureRandom.java:224)
        at com.sap.conn.rfc.engine.GUID.<clinit>(GUID.java:38)
. Try avoiding to initialize the class that caused initialization of the object. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
        at com.oracle.svm.hosted.image.DisallowedImageHeapObjectFeature.error(DisallowedImageHeapObjectFeature.java:173)
        at com.oracle.svm.core.image.DisallowedImageHeapObjects.check(DisallowedImageHeapObjects.java:62)
        at com.oracle.svm.hosted.image.DisallowedImageHeapObjectFeature.replacer(DisallowedImageHeapObjectFeature.java:149)
        at com.oracle.graal.pointsto.meta.AnalysisUniverse.replaceObject(AnalysisUniverse.java:582)
        at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.replaceObject(AnalysisConstantReflectionProvider.java:257)
        at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.interceptValue(AnalysisConstantReflectionProvider.java:228)
        at com.oracle.svm.hosted.heap.SVMImageHeapScanner.transformFieldValue(SVMImageHeapScanner.java:126)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.onFieldValueReachable(ImageHeapScanner.java:331)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.onFieldValueReachable(ImageHeapScanner.java:310)
        at com.oracle.graal.pointsto.heap.ImageHeapScanner.lambda$computeTypeData$1(ImageHeapScanner.java:153)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        ... 10 more
```


When i build with the following additional-build-args in my application.properties
`"--initialize-at-run-time=com.sap.conn.rfc.engine.GUID"`
I got this error
```
Error: Unsupported features in 2 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.sap.jdsr.writer.DsrIPassport. This error is reported at image build time because class com.sap.conn.jco.rt.ClientConnection is registered for linking at image build time by command line
Trace:
        at parsing com.sap.conn.jco.rt.ClientConnection.beginCall(ClientConnection.java:263)
Call path from entry point to com.sap.conn.jco.rt.ClientConnection.beginCall():
        at com.sap.conn.jco.rt.ClientConnection.beginCall(ClientConnection.java:248)
        at com.sap.conn.jco.rt.ClientConnection.execute(ClientConnection.java:1043)
        at com.sap.conn.jco.rt.ClientConnection.execute(ClientConnection.java:940)
        at com.sap.conn.jco.rt.RfcDestination.execute(RfcDestination.java:1140)
        at com.sap.conn.jco.rt.RfcDestination.execute(RfcDestination.java:1114)
        at com.sap.conn.jco.rt.AbapFunction.execute(AbapFunction.java:302)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:207)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:96)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.licenseToken(ConsoleProcessor.java:950)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.update(ConsoleProcessor.java:625)
        at com.zapliance.continuous.service.CownloadService.lambda$createAndStartCownloadTask$0(CownloadService.java:124)
        at com.zapliance.continuous.service.CownloadService$$Lambda$1400/0x00000007c3804b58.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:833)
        at com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:704)
        at com.oracle.svm.core.windows.WindowsPlatformThreads.osThreadStartRoutine(WindowsPlatformThreads.java:143)
        at com.oracle.svm.core.code.IsolateEnterStub.WindowsPlatformThreads_osThreadStartRoutine_77cb625316abfaaa1a6261614dc5eb03b33d1f85(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.sap.util.monitor.jarm.TaskMonitor. This error is reported at image build time because class com.sap.conn.jco.util.Jarm is registered for linking at image build time by command line
Trace:
        at parsing com.sap.conn.jco.util.Jarm.getRequestMonitor(Jarm.java:69)
Call path from entry point to com.sap.conn.jco.util.Jarm.getRequestMonitor(String, String):
        at com.sap.conn.jco.util.Jarm.getRequestMonitor(Jarm.java:67)
        at com.sap.conn.jco.rt.AbapRepository.queryFunctionTemplate(AbapRepository.java:921)
        at com.sap.conn.jco.rt.AbapRepository.queryFunctionTemplate(AbapRepository.java:549)
        at com.sap.conn.jco.rt.AbapRepository.getFunctionTemplate(AbapRepository.java:775)
        at com.sap.conn.jco.rt.BasicRepository.getFunction(BasicRepository.java:152)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:162)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:96)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.licenseToken(ConsoleProcessor.java:950)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.update(ConsoleProcessor.java:625)
        at com.zapliance.continuous.service.CownloadService.lambda$createAndStartCownloadTask$0(CownloadService.java:124)
        at com.zapliance.continuous.service.CownloadService$$Lambda$1400/0x00000007c3804b58.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:833)
        at com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:704)
        at com.oracle.svm.core.windows.WindowsPlatformThreads.osThreadStartRoutine(WindowsPlatformThreads.java:143)
        at com.oracle.svm.core.code.IsolateEnterStub.WindowsPlatformThreads_osThreadStartRoutine_77cb625316abfaaa1a6261614dc5eb03b33d1f85(generated:0)

com.oracle.svm.core.util.UserError$UserException: Unsupported features in 2 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.sap.jdsr.writer.DsrIPassport. This error is reported at image build time because class com.sap.conn.jco.rt.ClientConnection is registered for linking at image build time by command line
Trace:
        at parsing com.sap.conn.jco.rt.ClientConnection.beginCall(ClientConnection.java:263)
Call path from entry point to com.sap.conn.jco.rt.ClientConnection.beginCall():
        at com.sap.conn.jco.rt.ClientConnection.beginCall(ClientConnection.java:248)
        at com.sap.conn.jco.rt.ClientConnection.execute(ClientConnection.java:1043)
        at com.sap.conn.jco.rt.ClientConnection.execute(ClientConnection.java:940)
        at com.sap.conn.jco.rt.RfcDestination.execute(RfcDestination.java:1140)
        at com.sap.conn.jco.rt.RfcDestination.execute(RfcDestination.java:1114)
        at com.sap.conn.jco.rt.AbapFunction.execute(AbapFunction.java:302)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:207)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:96)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.licenseToken(ConsoleProcessor.java:950)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.update(ConsoleProcessor.java:625)
        at com.zapliance.continuous.service.CownloadService.lambda$createAndStartCownloadTask$0(CownloadService.java:124)
        at com.zapliance.continuous.service.CownloadService$$Lambda$1400/0x00000007c3804b58.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:833)
        at com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:704)
        at com.oracle.svm.core.windows.WindowsPlatformThreads.osThreadStartRoutine(WindowsPlatformThreads.java:143)
        at com.oracle.svm.core.code.IsolateEnterStub.WindowsPlatformThreads_osThreadStartRoutine_77cb625316abfaaa1a6261614dc5eb03b33d1f85(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.sap.util.monitor.jarm.TaskMonitor. This error is reported at image build time because class com.sap.conn.jco.util.Jarm is registered for linking at image build time by command line
Trace:
        at parsing com.sap.conn.jco.util.Jarm.getRequestMonitor(Jarm.java:69)
Call path from entry point to com.sap.conn.jco.util.Jarm.getRequestMonitor(String, String):
        at com.sap.conn.jco.util.Jarm.getRequestMonitor(Jarm.java:67)
        at com.sap.conn.jco.rt.AbapRepository.queryFunctionTemplate(AbapRepository.java:921)
        at com.sap.conn.jco.rt.AbapRepository.queryFunctionTemplate(AbapRepository.java:549)
        at com.sap.conn.jco.rt.AbapRepository.getFunctionTemplate(AbapRepository.java:775)
        at com.sap.conn.jco.rt.BasicRepository.getFunction(BasicRepository.java:152)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:162)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:96)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.licenseToken(ConsoleProcessor.java:950)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.update(ConsoleProcessor.java:625)
        at com.zapliance.continuous.service.CownloadService.lambda$createAndStartCownloadTask$0(CownloadService.java:124)
        at com.zapliance.continuous.service.CownloadService$$Lambda$1400/0x00000007c3804b58.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:833)
        at com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:704)
        at com.oracle.svm.core.windows.WindowsPlatformThreads.osThreadStartRoutine(WindowsPlatformThreads.java:143)
        at com.oracle.svm.core.code.IsolateEnterStub.WindowsPlatformThreads_osThreadStartRoutine_77cb625316abfaaa1a6261614dc5eb03b33d1f85(generated:0)

        at com.oracle.svm.core.util.UserError.abort(UserError.java:84)
        at com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:249)
        at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:754)
        at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:558)
        at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:515)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:407)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:585)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:128)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus.main(NativeImageGeneratorRunner.java:615)
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Unsupported features in 2 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.sap.jdsr.writer.DsrIPassport. This error is reported at image build time because class com.sap.conn.jco.rt.ClientConnection is registered for linking at image build time by command line
Trace:
        at parsing com.sap.conn.jco.rt.ClientConnection.beginCall(ClientConnection.java:263)
Call path from entry point to com.sap.conn.jco.rt.ClientConnection.beginCall():
        at com.sap.conn.jco.rt.ClientConnection.beginCall(ClientConnection.java:248)
        at com.sap.conn.jco.rt.ClientConnection.execute(ClientConnection.java:1043)
        at com.sap.conn.jco.rt.ClientConnection.execute(ClientConnection.java:940)
        at com.sap.conn.jco.rt.RfcDestination.execute(RfcDestination.java:1140)
        at com.sap.conn.jco.rt.RfcDestination.execute(RfcDestination.java:1114)
        at com.sap.conn.jco.rt.AbapFunction.execute(AbapFunction.java:302)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:207)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:96)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.licenseToken(ConsoleProcessor.java:950)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.update(ConsoleProcessor.java:625)
        at com.zapliance.continuous.service.CownloadService.lambda$createAndStartCownloadTask$0(CownloadService.java:124)
        at com.zapliance.continuous.service.CownloadService$$Lambda$1400/0x00000007c3804b58.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:833)
        at com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:704)
        at com.oracle.svm.core.windows.WindowsPlatformThreads.osThreadStartRoutine(WindowsPlatformThreads.java:143)
        at com.oracle.svm.core.code.IsolateEnterStub.WindowsPlatformThreads_osThreadStartRoutine_77cb625316abfaaa1a6261614dc5eb03b33d1f85(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.sap.util.monitor.jarm.TaskMonitor. This error is reported at image build time because class com.sap.conn.jco.util.Jarm is registered for linking at image build time by command line
Trace:
        at parsing com.sap.conn.jco.util.Jarm.getRequestMonitor(Jarm.java:69)
Call path from entry point to com.sap.conn.jco.util.Jarm.getRequestMonitor(String, String):
        at com.sap.conn.jco.util.Jarm.getRequestMonitor(Jarm.java:67)
        at com.sap.conn.jco.rt.AbapRepository.queryFunctionTemplate(AbapRepository.java:921)
        at com.sap.conn.jco.rt.AbapRepository.queryFunctionTemplate(AbapRepository.java:549)
        at com.sap.conn.jco.rt.AbapRepository.getFunctionTemplate(AbapRepository.java:775)
        at com.sap.conn.jco.rt.BasicRepository.getFunction(BasicRepository.java:152)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:162)
        at com.zapliance.modules.duniverseCownload.SAPTableQuerySimple.queryTable(SAPTableQuerySimple.java:96)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.licenseToken(ConsoleProcessor.java:950)
        at com.zapliance.modules.duniverseCownload.ConsoleProcessor.update(ConsoleProcessor.java:625)
        at com.zapliance.continuous.service.CownloadService.lambda$createAndStartCownloadTask$0(CownloadService.java:124)
        at com.zapliance.continuous.service.CownloadService$$Lambda$1400/0x00000007c3804b58.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:833)
        at com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:704)
        at com.oracle.svm.core.windows.WindowsPlatformThreads.osThreadStartRoutine(WindowsPlatformThreads.java:143)
        at com.oracle.svm.core.code.IsolateEnterStub.WindowsPlatformThreads_osThreadStartRoutine_77cb625316abfaaa1a6261614dc5eb03b33d1f85(generated:0)

        at com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:129)
        at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:751)
        ... 6 more
```




### Expected behavior

After adding the --initialize-at-run-time=com.sap.conn.rfc.engine.GUID it should work i guess

### Actual behavior

_No response_

### How to Reproduce?

I can give access to the private repository if really needed to selected ppl.

### Output of `uname -a` or `ver`

none

### Output of `java -version`

openjdk version "17.0.3" 2022-04-19 OpenJDK Runtime Environment GraalVM CE 22.1.0 (build 17.0.3+7-jvmci-22.1-b06) OpenJDK 64-Bit Server VM GraalVM CE 22.1.0 (build 17.0.3+7-jvmci-22.1-b06, mixed mode, sharing)

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.10.2.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.4

### Additional information

_No response_

--- a/docs/src/main/asciidoc/writing-native-applications-tips.adoc
+++ b/docs/src/main/asciidoc/writing-native-applications-tips.adoc
@@ -303,7 +303,23 @@ method com.amazonaws.services.s3.model.CryptoConfiguration.<init>(CryptoMode)
 Call path from entry point to com.amazonaws.services.s3.model.CryptoConfiguration.<init>(CryptoMode):
 ----
 
-If you need to delay the initialization of a class, you can use the `--initialize-at-run-time=<package or class>` configuration knob.
+Another common source of errors is when the image heap taken by GraalVM contains a `Random`/`SplittableRandom` instance:
+
+[source]
+----
+Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.
+----
+
+Which is more often than not caused by Quarkus initializing at build time a class with a static `Random`/`SplittableRandom` field,
+causing this particular instance to be tentatively included in the image heap.
+
+[TIP]
+====
+You can find detailed information about this `Random`/`SplittableRandom` issue in https://foivos.zakkak.net/tutorials/working-with-randoms-native-images/[this blog post].
+====
+
+In these cases, delaying the infringing class initialization at runtime might be the solution and, to achieve that,
+you can use the `--initialize-at-run-time=<package or class>` configuration knob.
 
 It should be added to the `native-image` configuration using the `quarkus.native.additional-build-args` configuration property as shown in the examples above.
 

